tests/manual_verification.py

```
@@ -0,0 +1,161 @@
+    config = RedemptionCodeConfig()
+
+    # 保存测试数据
+    test_codes = ['TEST001', 'TEST002', 'TEST003']
+    print(f"保存兑换码: {test_codes}")
+    config.codes_list = test_codes
+
+    # 重新加载
+    loaded_codes = config.codes_list
+    print(f"加载兑换码: {loaded_codes}")
+
+    assert loaded_codes == test_codes, f"期望 {test_codes}，实际 {loaded_codes}"
```

Ruff (0.14.14)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)